### ISSUE TITLE
Anyone, No-one Sensor

### ISSUE THREAD

**Issue by samsonrosen** [PeteLawrence/homebridge-people#29]
Hey!

I've installed this plugin and it works really well. I created static IPs for the four iPhones that regularly come in and out the house, and they are being triggered with the occupancy sensor.

In my config I have enabled both sensors:

    ```{
         "accessory":"people",
         "name":"People",
         "people":[
            {
               "name":"John1",
               "target":"192.168.1.86"
            },
            {
               "name":"John2",
               "target":"192.168.1.217"
            },
            {
               "name":"John3",
               "target":"192.168.1.125"
            },
            {
               "name":"John4",
               "target":"192.168.1.115"
            }
         ],
        "threshold":15,
	"anyone_sensor":true,
        "noone_sensor":true
      }```

However, I do not see a separate 'no-one sensor'. Just an 'anyone_sensor'. Am I misunderstanding how this functionality works? Or is it the act of no one being in the house the 'Anyone' sensor inactive?
![img_1391](https://cloud.githubusercontent.com/assets/7211878/21166598/6f54cd30-c19e-11e6-81fc-1af2e226629c.PNG)


Thanks,

Sam

**Comment by sbels** [comment]
I am getting the same thing even when setting noone to true in config.

**Comment by erfinb** [comment]
Do you all have to check your app in order for it to check the status of people? Or is it doing this in the background for you? And if you open the index file in the homebridge-people plugin and then set no one from false to true it will show up.

**Comment by creedda** [comment]
Looks like the documentation is wrong. Make sure you capitalize as shown below.

the settings should be: "anyoneSensor": true,
  "nooneSensor": true

**Comment by samsonrosen** [comment]
Thanks @creedda, this worked! The NoOne sensor shows up now.

**Comment by benzman81** [comment]
Documentation will be fixed with pull request "Major Rework  #35".
